tutorials/i18n/internationalizing_games.rst

```diff
@@ -58,6 +58,42 @@ Select the resource to be remapped then add some alternatives for each locale.
     the current language, making it ideal for things like multiplayer chat where
     the text language may not match the client's language.
 
+Automatically setting a language
+--------------------------------
+It is recommended to default to the user's preferred language which can be obtained via :ref:`OS.get_locale_language() <class_OS_method_get_locale_language>`. If your game is not available in that language, it will fall back to the one in **Project settings > General > Internationalization > Locale (Advanced settings)** (if empty `en` for English will be used which is recommended).
+Nevertheless letting players change the language in game is recommended for various reasons (e.g. translation quality or player preference).
+
+.. tabs::
+ .. code-tab:: gdscript
+
+    var language = "automatic"
+    # Load here language from the user settings file
+    if language == "automatic":
+       var preferred_language = OS.get_locale_language()
+       TranslationServer.set_locale(preferred_language)
+    else:
+       TranslationServer.set_locale(language)
+
+Locale vs. language
+-------------------
+A Locale is commonly a combination of a language with a region or country but can also contain more information like scripts.
```

Comment:
```suggestion
A :ref:`locale <doc_locales>` is commonly a combination of a language with a region or country, but can also contain information like a script or a variant.
```
No need to capitalize locale here
Slightly clarified the sentence as well
Linked to the list of locales, too